src/main/java/service/ParserOperation.java

```diff
@@ -0,0 +1,7 @@
+package service;
```

Comment:
You must create 5 separeted services
-read from file
-data parser service
-calculator
-report maker
-write to file